framework/src/Volo.Abp.AspNetCore.Mvc.Client/Volo/Abp/AspNetCore/Mvc/Client/MvcRemoteTenantStore.cs

```diff
@@ -79,6 +80,11 @@ public class MvcRemoteTenantStore : ITenantStore, ITransientDependency
         return tenantConfiguration?.Value;
     }
 
+    public Task<IReadOnlyList<TenantConfiguration>> GetListAsync(bool includeDetails = false)
+    {
```

Comment:
https://github.com/abpframework/abp/pull/19208/files#r1512188993

Reply:
That would be a problem on the security side; otherwise, we should add permission check for the user to have tenant list access. Since all application uses the tenant management module [TenantStore](https://github.com/abpframework/abp/blob/8ba6796c06b46d21ff1cff5f54ed07cba607bb55/modules/tenant-management/src/Volo.Abp.TenantManagement.Domain/Volo/Abp/TenantManagement/TenantStore.cs#L11) excepts mvc client and maui blazor (which we don't need this method on client applications).